packages/demand-deeplink/src/errors.ts

```diff
@@ -28,6 +30,8 @@ export const errorMessageByCode = {
   [codes.DP010]: `Unsupported booking type. Supported types are ${BookingTypes.ASAP} and ${BookingTypes.PREBOOK}`,
   [codes.DP011]: `Booking type is ${BookingTypes.ASAP} but pickup time is specified`,
   [codes.DP012]: 'Incorrect coordinates value',
+  [codes.DP013]: 'Unacceptable parameter',
```

Comment:
🤔 can we specify why it's Unacceptable ?

Reply:
We can rename this error like "Pickup or dropoff parameters are unacceptable"